src/OpenTracing/OpenTracing.csproj

```diff
@@ -6,6 +6,9 @@
 
 For the time being, mild backwards-incompatible changes may be made without changing the major version number. As OpenTracing and opentracing-csharp mature, backwards compatibility will become more of a priority.</Description>
     <PackageTags>opentracing;distributed-tracing;tracing;logging</PackageTags>
+    
+    <DocumentationFile>bin\$(Configuration)\netstandard1.0\OpenTracing.xml</DocumentationFile>
```

Comment:
This is already configured in Directory.Build.props but I only enabled it for Release-builds. I guess that's an unnecessary optimization so if you want, you can remove the `and '$(Configuration)'=='Release'` from there. I can also do it after this PR has been merged since it's not a blocker.

Reply:
I fixed this in the master-branch. xml files are now always created for all projects.